assets/migrations/postgres/006_add_collate_index.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose NO TRANSACTION
+-- +goose Up
+CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_collate on tuple (
+    store,
+    _user,
+    relation,
+    object_type,
+    object_id collate "C"
+);
+
+DROP INDEX IF EXISTS idx_reverse_lookup_user;
+
+-- +goose Down
+DROP INDEX IF EXISTS idx_collate;
+
+CREATE INDEX idx_reverse_lookup_user on tuple (store, object_type, relation, _user);
```

Comment:
```suggestion
CREATE INDEX idx_reverse_lookup_user on tuple (store, object_type, relation, _user);

```
nit: please add a trailing newline